docs/pages/admin-guides/access-controls/access-requests/access-request-configuration.mdx

```diff
@@ -294,6 +294,44 @@ user:
 1. If one of the user's roles includes the `reason` strategy, the user must
    provide a reason when authenticating.
 
+## Reason mode
+
+Reason mode allows enforcing users to provide reason while making an Access Request. It only works
+in allow rules and is set with `allow.request.reason.mode`.
```

Comment:
nit: "Reason mode" feels weird written out in english, maybe this is better
```suggestion
## Requiring request reasons

The `allow.request.reason.mode` field controls whether a reason is required when users submit access requests.
```

Reply:
I don't have a strong opinion here. @r0mant ?

Reply:
Nic's suggestion looks good to me.

Reply:
14574945c841dd281e458f52c6c4486612bc12db